src/App.tsx

```diff
@@ -16,36 +18,72 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortType, setSortType] = useState<SortType>(SortType.None);
```

Comment:
In **TypeScript**, when you use `useState` with an initial value like `SortType.None`, **TypeScript is able to infer** the type of `sortType` automatically based on the initial value. So, you don't always need to specify `<SortType>` if the initial value already makes it clear what the type should be.

```suggestion
  const [sortType, setSortType] = useState(SortType.None);
```